front-dispatcher/src/App.tsx

```diff
@@ -8,6 +9,7 @@ function App() {
 
   return (
     <div className="App">
+      <GlobalStyle />
```

Comment:
if you are using global style you need to wrap the content with it
<GlobalStyle>{......}</GlobalStyle>